types/index.d.ts

```diff
@@ -112,3 +112,49 @@ export interface FunctionOutput extends Record<string, unknown> {
      */
     name: string;
 }
+
+export type RetryOptions = FixedDelayRetryOptions | ExponentialBackoffRetryOptions;
+
+export interface FixedDelayRetryOptions {
+    /**
+     * A specified amount of time is allowed to elapse between each retry.
+     */
+    strategy: 'fixedDelay';
+
+    /**
+     * The maximum number of retries allowed per function execution. -1 means to retry indefinitely.
+     */
+    maxRetryCount: number;
+
+    /**
+     * The delay that's used between retries.
+     * This can be a Node.js Date or Unix time in milliseconds.
+     */
+    delayInterval: Date | number;
```

Comment:
That's a good point. I was kind of copying existing support we had for the "expires" property on http cookies, but that's an actual timestamp not a duration. I'll have to look into this

Reply:
Updated to use a new duration interface instead of `Date`. I considered supporting the string "HH:mm:ss", but felt like an object was simpler for me to implement (no string parsing) and probably more type-safe/readable for users

Reply:
Yeah that's fair, although I would still argue it's better to be consistent with our docs, and perhaps there's a package that can already do the parsing for us? Regardless, I won't block on this

Reply:
We have to update the docs regardless for the v4 model samples, so shouldn't be a concern.

I prefer the interface over the string anyways so not sure how relevant it is, but yes there are packages to parse date/time strings (dayjs/luxon/moment). They have their own problems and it's not always worth adding packages even if we preferred that approach, though.